lib/timeline/component/Group.js

```diff
@@ -486,7 +484,7 @@ Group.prototype._updateSubgroupsSizes = function () {
   if (me.subgroups) {
     for (var subgroup in me.subgroups) {
       var newStart = me.subgroups[subgroup].items[0].data.start;
-      var newEnd = me.subgroups[subgroup].items[0].data.end;
+      var newEnd = me.subgroups[subgroup].items[0].data.end - 1;
 
       me.subgroups[subgroup].items.forEach(function(item) {
         if (new Date(item.data.start) < new Date(newStart)) { 
```

Comment:
I agree. But because we allow the user to add strings to the `start` and `end` of items, we might encounter comparisons of strings, which is very not good. So as a precaution and to keep consistency in the timeline, Dates are always used for comparisons. I think a better way would have been using `Moment` but This works too